docs/build/p2p-node/index.md

```diff
@@ -161,109 +104,95 @@ $ subkey --sr25519 generate
+--ipfsPinningServiceName pinata \\
+--ipfsPinningServiceURL <pinata endpoint> \\
+--ipfsPinningServiceAuth <your pinata auth token> \\
+--podOperatorSecretSeed <secret seed for POD operator> \\
+--podAdminSecretSeed <secret seed for POD admin> \\
 ```
 
-Replace the `<KEY-FILE>` with the key file you obtained when creating the Ethereum account and `<DEFINE_CONFIG_DIR_NAME>` with the location where you want the `config.yaml` file to be stored. Note that the target direction -t should be specified with an absolute path. Manually add `https://` to the Infura link (see above).
+**NOTE**:
 
-The password for the provided `keystore/<KEY-FILE>` file is asked once the `createconfig` command is run. If the password is not set, just press `enter`.
+- **The generated `config.yaml` includes sensitive information regarding the accounts used to authenticate and sign transactions. Make sure to store it in a secure environment.**
 
-**NOTE**: The generated `config.yaml` includes information about your Ethereum private key. Make sure to store it in a secure environment.
 
-### Set up your Centrifuge Node config.yaml for the corresponding testnet or mainnet
+- `podOperatorSecretSeed` - if this is omitted a new secret seed will be generated by the node, please see [POD operator](#pod-operator) for more information regarding this account.
 
-If you want to switch between testnets and mainnet, adjust the following accordingly and add the corresponding Infura link (see above). Manually add `https://` to the Infura link.
 
-**Networks:**
+- `podAdminSecretSeed` - if this is omitted a new secret seed will be generated by the node, please see [POD admin](#pod-admin) and [token usage](#usage) for more information regarding this account.
 
-- **Kovan - Catalyst**
 
-Use network `-n catalyst`
+- For more information regarding IPFS pinning, please see [IPFS](#ipfs).
 
-- **Mainnet**
+#### Network Configurations
 
-Use network `-n mainnet`
+[//]: # (TODO: Update POD link)
 
----
+Besides `mainnet`, Centrifuge has support for the `catalyst` test network. The network configuration for the different testnets is also part of the [code base](https://github.com/centrifuge/go-centrifuge/blob/master/build/configs/default_config.yaml). This enables the client user to run on top of them with minimum configuration needed. Please find the most important information summarized below:
 
-**If you like to run the Centrifuge node with your own Ethereum or Centrifuge Ehain nod,e please replace the Infura urls with your own `Geth` and `centrifuge-chain` Node-URL.**
+##### Catalyst
 
----
+Use network `-n catalyst`.
 
-### Look up your Centrifuge ID
+This network is a test network running a version of the Centrifuge Chain modified for testing.
 
-A participant of the Centrifuge P2P network will be identified by an identity (Centrifuge ID) within the network. The `createconfig` command automatically creates an identity and the key pairs.
+- Client: parity
+- Purpose: Testnet
+- Bootstrap Nodes:
```

Comment:
These bootstrap nodes is referring to the offchain p2p bootnodes and not the chain nodes. I do not think we stood up any bootnodes yet, so for now I would keep it empty (or ask our dev team in the DAO channel).